### ISSUE TITLE
Tests not starting when jobs from previous runs are still on namespace in Openshift

### ISSUE THREAD

**Issue by jah755** [citrusframework/yaks#466]
I'm running yaks tests through task in tekton pipeline. First run succeeds and performed jobs are present on a namespace:
![image](https://user-images.githubusercontent.com/112621000/220361620-ce5009c2-107d-42f0-83ca-c5dce3f8728a.png)
In the next run of pipeline tests are not being performed:
![image](https://user-images.githubusercontent.com/112621000/220361857-9539be5a-e096-4722-8d33-46c0c6aa69db.png)
I noticed that after manually removing existing completed jobs tests are performed again. 
Is this an expected behaviour?

**Comment by christophd** [comment]
No, not expected behavior. I think this is an issue with YAKS related to Tekton pipelines and I have to admit that Tekton pipeline integration for YAKS has not been a focus, yet.

Could you please provide the pipeline details and how you integrate the YAKS tests in Tekton? Thx
